scripts/get-model-filename

```diff
@@ -21,15 +21,17 @@ will be created if it does not exist.
   Copied: /tmp/myfile.json -> /Users/foo/botocore/data/aws/cloudwatch/2010-08-01.normal.json
 
 """
+import json
+
 # Note we're using optparse for 2.6 compat.
```

Comment:
Hm, the comment around the import causing a space is a little strange. I don't feel too strongly on this though... thoughts? Also, did we mean to hit the `scripts` directory as well?

Reply:
Yeah, black may clean this up later, but I don't think we have a way to disable it. I'm also relatively indifferent on it for the moment. I don't think it particularly hurts readability.

For scripts, we were already including it as part of the .pre-commit hooks. It seems reasonable to keep those files formatted too, but it's not strictly required.